docs/extras/blocking_updates.md

```diff
@@ -1,12 +1,35 @@
 # Blocking Updates
 
-You can add a custom DNS to your WiFi connection that will block all communication with Nintendo's servers. This will stop any updates and reporting to Nintendo, but it will prevent use of the eShop and online games.
+This page will help you set up a method to block Nintendo from reaching your switch. This will stop any updates and reporting to Nintendo, but it will prevent use of the eShop and online games.
```

Comment:
Not sure if I like `block Nintendo from reaching your switch`. Why not stick with `block all communication with Nintendo's servers`?